src/tests/signup.test.js

```diff
@@ -22,11 +22,11 @@ describe('Testing Signup Route with errors', function () {
       username: 'testish111',
       password,
     };
-    await userServices.createUser(testUser);
+    await User.create(testUser);
```

Comment:
why remove userService.create? if we were to change the name of the model from the User will you go through all files updating the user model?